src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -427,6 +435,9 @@ public void removeDataBundle(DataBundle dataBundle) {
             }
         }
         accountsDb.deleteAccounts(dataBundle.accounts.values());
+
+        // Delete all admin emails
```

Comment:
I suggested lower case because there's another comment in this commit that starts with a lower case. Yes, seems our coding standard uses upper case. So can keep this as it is.
I checked [Google coding standard](https://google.github.io/styleguide/javaguide.html#s7-javadoc) and even they are not consistent 😛 
Always starting with an upper case letter seems like an easier rule to follow (as opposed to `lower case for phrases, upper case for sentences`)